src/xcode/ENA/ENAUITests/HealthCertificate/CreateHealthCertificate.swift

```diff
@@ -40,6 +42,7 @@ class CreateHealthCertificate: XCTestCase {
 	}
 
 	func test_CreateAntigenTestProfileWithFirstCertificate_THEN_DeleteProfile() throws {
+		app.launchArguments.append(contentsOf: ["-noHealthCertificate", YES])
```

Comment:
Is this `YES` alias still active?
```suggestion
		app.launchArguments.append(contentsOf: ["-noHealthCertificate", "YES"])
```